app/components/SidebarHeader.tsx

```
@@ -0,0 +1,246 @@
+  handleCloseSidebar,
+  isCollapsed,
+  isMobileOverlay = false,
+}) => {
+  const { user } = useAuth();
+  const isMobile = useIsMobile();
+  const router = useRouter();
+  
+  // Only use useSidebar when not in mobile overlay
+  let toggleSidebar = () => {};
+  if (!isMobileOverlay) {
+    const { toggleSidebar: sidebarToggle } = useSidebar();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)